src/include/basil.h

```diff
@@ -319,17 +358,48 @@
 #define BASIL_VAL_IN		"IN"		/* BASIL_ATR_ACTION */
 #define BASIL_VAL_OUT		"OUT"		/* BASIL_ATR_ACTION */
 
+/* if set, the specified env var is the href (i.e., url) of an xslt file */
+#define BASIL_XSLT_HREF_ENV	"BASIL_XSLT_HREF"
 
-/* BASIL supported versions array */
+/*
+ * BASIL versions.
+ * To add a new version, define the BASIL_VAL_VERSION_#_# string, above,
+ * then place it at the top of the supported_versions array.
+ * Define a numeric version value in the enum, below.
+ * Add the strcmp to match the string to the enum value in
+ * request_start() in handlers.c
+ */
+
+/* BASIL supported versions string array */
 
 /* The first version listed is considered the current version. */
 static const char *basil_supported_versions[] __attribute__((unused)) = {
+	BASIL_VAL_VERSION_1_4,
+	BASIL_VAL_VERSION_1_3,
 	BASIL_VAL_VERSION_1_2,
 	BASIL_VAL_VERSION_1_1,
 	BASIL_VAL_VERSION_1_0,
-	(char *) 0
+    (char *) 0
 };
 
+/**
+ * BASIL versions -- numerical
+ */
+typedef enum {
+	basil_1_1 = 11,
```

Comment:
If you are enumerating all available versions, why not start with 1.0? Supported versions are defined in alps.c.